core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -653,7 +653,9 @@ private[spark] object SparkConf extends Logging {
     "spark.memory.offHeap.enabled" -> Seq(
       AlternateConfig("spark.unsafe.offHeap", "1.6")),
     "spark.rpc.message.maxSize" -> Seq(
-      AlternateConfig("spark.akka.frameSize", "1.6"))
+      AlternateConfig("spark.akka.frameSize", "1.6")),
+    "spark.yarn.jars" -> Seq(
+      AlternateConfig("spark.yarn.jar", "2.0"))
```

Comment:
I'm probably wrong about this, but is it `.archive` that is the new name for `.jar` rather than `.jars`?